.travis.yml

```diff
@@ -14,7 +14,6 @@ install:
   - sudo chmod +x /usr/local/bin/src
   - src toolchain add sourcegraph.com/sourcegraph/srclib-python
   - go get -d -v ./... && go build -v ./...
-  - make test-dependencies
```

Comment:
- [ ] Can you explain why this was removed? Do we not use these repos in our tests?